crypto/x509/x509_acert.c

```diff
@@ -326,3 +326,76 @@ const STACK_OF(X509_EXTENSION) *X509_ACERT_get0_extensions(const X509_ACERT *x)
 {
     return x->acinfo->extensions;
 }
+
+int X509_verify_acert(X509_STORE_CTX *ctx, X509_ACERT *acert)
+{
+    return 0; /* Stub implementation. To be replaced. */
+}
+
+OSSL_ISSUER_SERIAL *X509_get_issuer_serial(X509 *x)
```

Comment:
Does a standard certificate have an ISSUER_SERIAL?
Or is this meant to be against an ACERT?

It is unfortunate that we have OSSL_ISSUER_SERIAL and we have a standard certificate serial and we have APIs like OSSL_STORE_SEARCH_by_issuer_serial which are fun for the user to keep straight.

Reply:
I meant to include one for attribute certificates too, but I did intend this for public key certificates. The public key certificate does not have a field of type `IssuerSerial` per se, but that field is a simple composite of the `issuer` field and the `serialNumber` fields of the public key certificate. This is particularly important for attribute certificates because one of the way that attributes can be associated with a particular end-entity is via the issuer and serial number.

For reference, this type is used in the `holder` field in the attribute certificate definition:

```asn1
Holder ::= SEQUENCE {
  baseCertificateID  [0]  IssuerSerial OPTIONAL,
  entityName         [1]  GeneralNames OPTIONAL,
  objectDigestInfo   [2]  ObjectDigestInfo OPTIONAL }
  (WITH COMPONENTS {..., baseCertificateID  PRESENT } |
   WITH COMPONENTS {..., entityName  PRESENT } |
   WITH COMPONENTS {..., objectDigestInfo  PRESENT } )

IssuerSerial ::= SEQUENCE {
  issuer     GeneralNames,
  serial     CertificateSerialNumber,
  issuerUID  UniqueIdentifier OPTIONAL,
  ... }
```

Reply:
Oh, nevermind, I see I have `X509_ACERT_get0_holder_baseCertId` which is the attribute certificate equivalent. It is named that way because of an `IssuerSerial` can also occur in the `issuer` field of an attribute certificate, so we need something in the name to distinguish where it comes from.

Reply:
I will mark this as resolved, since I believe I answered the question and no code changes are needed.